package.json

```diff
@@ -15,8 +15,11 @@
   "dependencies": {
     "@nuxtjs/firebase": "^7.6.1",
     "@ourjapanlife/findadoc-localization": "https://github.com/ourjapanlife/findadoc-localization",
+    "@nuxtjs/recaptcha": "^1.0.4",
+    "axios": "^0.21.1",
```

Comment:
Someone else is actually adding the `@nuxt/http` library  in another PR which would be used instead of axios. 😞 sorry, but can you switch to using that instead when you get a chance?